src/PPU.cpp

```diff
@@ -163,14 +134,14 @@ void PPU::updateTileData() {
 			uint8_t lo2 = bus->read(block2_start + i * 16 + j * 2);
 			uint8_t hi2 = bus->read(block2_start + i * 16 + j * 2 + 1);
 
-			set_line(block0, x * 8, j + y * 8, hi0, lo0);
-			set_line(block1, x * 8, j + y * 8, hi1, lo1);
-			set_line(block2, x * 8, j + y * 8, hi2, lo1);
+			set_line(x * 8, j + y * 8, hi0, lo0);
+			set_line(x * 8, 64 + j + y * 8, hi1, lo1);
+			set_line(x * 8, 128 + j + y * 8, hi2, lo2);
```

Comment:
Since I'm doing everything in one pixel buffer rather than 3 sprites I had to add an offset. I'm planning on simplifying this logic and clarifying it once I get back into PPU development.